refinery/main.py

```diff
@@ -838,6 +842,28 @@ def queue_add_all(self, e=None):
 
         tree_frame.activate_all()
 
+    def queue_add_all_maps(self, e=None):
```

Comment:
This is an issue for another PR, but what is e? That doesn't appear to be documented anywhere.

Reply:
I'm absolutely clueless on what e is in this context too